### PR TITLE
Allow Rawls to accept null entityType, entityName [WA-135]

### PR DESCRIPTION
**Pull request by aednichols**
Fixes a subtle regression introduced in https://github.com/broadinstitute/rawls/pull/1174

### CODE REVIEW

**Review by grsterin — APPROVED** [review]

**Review by rushtong — APPROVED** [review]
👍

**Comment by aednichols** [comment]
Thank you @rushtong !

File: core/src/main/scala/org/broadinstitute/dsde/rawls/model/ExecutionModel.scala [hunk 1]
```diff
@@ -375,12 +375,13 @@ class ExecutionJsonSupport extends JsonSupport {
         // so in that project one can just adjust the function call to jsonFormat*N*.
         methodConfigurationNamespace = fields("methodConfigurationNamespace").convertTo[String],
         methodConfigurationName = fields("methodConfigurationName").convertTo[String],
-        entityType = fields.get("entityType").map(_.convertTo[String]),
-        entityName = fields.get("entityName").map(_.convertTo[String]),
-        expression = fields.get("expression").map(_.convertTo[String]),
+        // We are flatMapping these because `null` is a valid input provided by the UI
```
Comment (breilly2) [inline core/src/main/scala/org/broadinstitute/dsde/rawls/model/ExecutionModel.scala:378]: Sorry I'm a little late to this, but maybe it's a bug that the UI is sending `null`s? I have not investigated anything yet but I can if you think there may be a UI problem, @aednichols .

Comment (aednichols) [reply core/src/main/scala/org/broadinstitute/dsde/rawls/model/ExecutionModel.scala:378]: We're talking the FireCloud UI, which didn't change within the timespan when the bug appeared

**Pull request merged**